config/config_test.go

```diff
@@ -3150,3 +3178,44 @@ func TestTCF2FeatureOneVendorException(t *testing.T) {
 		assert.Equal(t, tt.wantIsVendorException, value, tt.description)
 	}
 }
+
+func TestAccountPriceFloors_validate(t *testing.T) {
+	type fields struct {
+		Enabled                bool
+		EnforceFloorRate       int
+		AdjustForBidAdjustment bool
+		EnforceDealFloors      bool
+		UseDynamicData         bool
+	}
+	type args struct {
+		errs []error
+	}
```

Comment:
If we don't envision expanding the args list at this time, I suggest simplifying this by deleting this type and changing `args args` and `want []error` to something like
```
giveErrs []error
wantErrs []error
```

Reply:
updated